books/A-Game-Of-Thrones.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "A Game Of Thrones",
+  "storyline": "A Game of Thrones takes place over the course of one year on or near the fictional continent of Westeros. The story begins when King Robert visits the northern castle Winterfell to ask Ned Stark to be his right-hand assistant, or Hand of the King. The previous Hand, Jon Arryn, died under suspicious circumstances.",
+  "authors": ["George R.R. Martin"],
+  "genres": "Poilitical novel, Epic Fantasy",  
```

Comment:
@k1dnyy `genres` should have been an array.
Refer:
https://github.com/404minds/books-db/blob/a5db372f024697c05257a11e985ba88cfaa6df49/books/neuromancer.json#L5